docs/source/find_data/organizations/icgc.md

```diff
@@ -78,13 +78,12 @@ Written documents have also been scanned at 300/400 dpi.
 Each institution is in charge of physical preservation. See also:
 * [Ebre Obesrvatory](https://www.icgc.cat/en/Public-Administration-and-Enterprises/Services/Recorded-earthquakes-and-seismic-information/Seismic-information-and-maps-collections/Ebre-Observatory)
 * [Fabra Observatory](https://www.icgc.cat/en/Public-Administration-and-Enterprises/Services/Recorded-earthquakes-and-seismic-information/Seismic-information-and-maps-collections/Fabra-Observatory)
-* [Seismic Bulletins 1984 - ](https://www.icgc.cat/en/Public-Administration-and-Enterprises/Services/Recorded-earthquakes-and-seismic-information/Seismic-information-and-maps-collections/Seismological-Bulletins)
+* [Seismic Bulletins 1984 - ](https://www.icgc.cat/en/Public-Administration-and-Enterprises/Services/Recorded-earthquakes-and-seismic-information/Seismic-information-and-maps-collections/Seismological-Bulletins)<!--the above three links are broken and the homepage is directed to https://www.icgc.cat/en-->
```

Comment:
Ebre: https://www.icgc.cat/en/Thematic-areas/Risks-and-emergencies/Earthquakes/Seismic-information-and-maps-collections/Ebre-Observatory/Ebre-seismic-station-EBR/Search-available-analog-seismic-records-EBR-station
Fabra: https://www.icgc.cat/en/Thematic-areas/Risks-and-emergencies/Earthquakes/Seismic-information-and-maps-collections/Fabra-Observatory/Seismic-records-Observatori-Fabra/Search-available-analog-seismic-records-FBR-station
Bulletins: https://www.icgc.cat/en/Seismological-Bulletins